scripts/post-generate.js

```
@@ -0,0 +1,164 @@
+  const transformed = {};
+  for (const [key, value] of Object.entries(obj)) {
+    const camelKey = toCamelCase(key);
+    transformed[camelKey] = transformProperties(value);
+  }
+  return transformed;
+}
+
+// Function to transform snake_case to camelCase in a string
+function transformSnakeToCamel(content) {
+  // Transform property names in interfaces and types
+  content = content.replace(/'([a-z_]+)':/g, (match, p1) => {
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)